### ISSUE TITLE
I have produced a web Quest editor

### ISSUE THREAD

**Issue by 233sama** [FlyingPikachu/Quests#122]
visit：
http://www.mclll.com/T/index_en.htm

ChineseVersion:
http://www.mclll.com/T/index.htm

You can use the editor to easily generate a complete Quest (probably not more than five minutes), from left to drag the icon to the right of the container, you can generate real-time reading the file.

Any Questions Please Contact Me 244541495@qq.com

**Comment by 233sama** [comment]
An OlderVersion：
http://www.mclll.com/taskeditor_en/

Older CHN Version
http://www.mclll.com/taskeditor/

**Comment by FlyingPikachu** [comment]
That's incredible! I will experiment with this some more and get back to you. Thanks for your hard work!

**Comment by whoshenry** [comment]
This is really great.

**Comment by FlyingPikachu** [comment]
Agreed. @233sama Is there any way the new English version could be fully translated? I would be happy to help with that.

**Comment by 233sama** [comment]
Please contact me to give me your email address, I send all the documents to you. There are also a lot of improvements that need to be improved, and I hope you make more suggestions and make this editor better.

**Comment by FlyingPikachu** [comment]
Github doesn't have private messaging, so feel free to contact me on the Spigot forums :)
